enums.py

```diff
@@ -340,6 +340,13 @@ class AttributeState(Enum):
     FAILED = "FAILED"
 
 
+class AttributeVisibility(Enum):
+    HIDE = "HIDE"
```

Comment:
maybe a comment for the enum that they are in order (or including or however we want to name it :D )
so basically that it's clear that HIDE_ON_LABELING_PAGE will also hide it on DATA_BROWSER